### PR TITLE
VZ 5802: Document Jaeger/Istio integration

### PR DESCRIPTION
**Pull request by anders-swanson**
Document configuration required to integrate Istio with Jaeger tracing.

### CODE REVIEW

File: content/en/docs/monitoring/tracing/jaeger-tracing.md [hunk 1]
```diff
@@ -103,3 +103,53 @@ To export traces, annotate your applications with the `"sidecar.jaegertracing.io
 You can view the UI by port forwarding the Jaeger query service or by configuring an ingress controller for HTTPS access.
 Explore the Jaeger configuration in more detail using the
 [Jaeger Custom Resource Documentation](https://www.jaegertracing.io/docs/1.33/operator/#configuring-the-custom-resource).
+
```
Comment (rosemarymarano) [inline content/en/docs/monitoring/tracing/jaeger-tracing.md:106]: On line 38: kubectl create secret generic jaeger-secret -> $ kubectl create secret generic jaeger-secret (dollar sign before command)

File: content/en/docs/monitoring/tracing/jaeger-tracing.md [hunk 2]
```diff
@@ -103,3 +103,53 @@ To export traces, annotate your applications with the `"sidecar.jaegertracing.io
...
+      istioInstallArgs:
+        - name: "meshConfig.enableTracing"
+          value: "true"
+```
+
+After enabling tracing, Istio will automatically configure itself with the Jaeger endpoint in your cluster, 
+and any new Istio-injected pods will begin exporting traces to Jaeger. Existing pods require a restart 
+to pull the new Istio configuration and start sending traces.
+
+By default, Istio samples traces at a 1% rate, meaning 1 in 100 requests will be traced in Jaeger.
```
Comment (rosemarymarano) [inline content/en/docs/monitoring/tracing/jaeger-tracing.md:135]: Istio samples traces -> Istio samples trace

File: content/en/docs/reference/API/Verrazzano/Verrazzano.md [hunk 3]
```diff
@@ -197,7 +197,8 @@ spec:
 | --- | --- | --- | --- | --- |
 | `gateways.istio-ingressgateway.externalIPs` | [NameValue](#name-value)  | string list | The external IP address used by the Istio ingress gateway. |  No |
 | `gateways.istio-ingressgateway.serviceAnnotations.*` | [NameValue](#name-value) | string | Annotations used for the Istio ingress gateway.  For sample usage, see [Customize Ingress](/docs/setup/customizing/ingress/). |  No |
-
+| `meshConfig.enableTracing` | [NameValue](#name-value)  | string | If `"true"`, Istio will export tracing when Jaeger is installed. Defaults to `"false"`. |  No |
```
Comment (rosemarymarano) [inline content/en/docs/reference/API/Verrazzano/Verrazzano.md:200]: `"true"`
`"false"`
next line, `"1"`
You don't need back ticks (code font) and double quotes; back ticks alone will denote the literal value.

**Review by rosemarymarano — COMMENTED** [review]
Minor edits

File: content/en/docs/monitoring/tracing/jaeger-tracing.md (hunk 2 above)
Comment (anders-swanson) [reply content/en/docs/monitoring/tracing/jaeger-tracing.md:135]: What about: "The default sampling rate is 1%"

File: content/en/docs/monitoring/tracing/jaeger-tracing.md [hunk 4]
```diff
@@ -103,3 +103,53 @@ To export traces, annotate your applications with the `"sidecar.jaegertracing.io
 You can view the UI by port forwarding the Jaeger query service or by configuring an ingress controller for HTTPS access.
 Explore the Jaeger configuration in more detail using the
 [Jaeger Custom Resource Documentation](https://www.jaegertracing.io/docs/1.33/operator/#configuring-the-custom-resource).
+
+
+## Configure the Istio mesh to use Jaeger Tracing
```
Comment (rosemarymarano) [inline content/en/docs/monitoring/tracing/jaeger-tracing.md:108]: Configure the Istio mesh to use Jaeger Tracing -> Configure the Istio mesh to use Jaeger tracing

File: content/en/docs/reference/API/Verrazzano/Verrazzano.md (hunk 3 above)
Comment (anders-swanson) [reply content/en/docs/reference/API/Verrazzano/Verrazzano.md:200]: The literal value is supposed to include the double quotes.

File: content/en/docs/monitoring/tracing/jaeger-tracing.md (hunk 2 above)
Comment (rosemarymarano) [reply content/en/docs/monitoring/tracing/jaeger-tracing.md:135]: Yes, that's fine.

File: content/en/docs/reference/API/Verrazzano/Verrazzano.md (hunk 3 above)
Comment (rosemarymarano) [reply content/en/docs/reference/API/Verrazzano/Verrazzano.md:200]: OH. My mistake.

**Review by rosemarymarano — APPROVED** [review]

**Pull request merged**